bugbug/bugzilla.py

```diff
@@ -137,12 +137,14 @@ def download_bugs(bug_ids, products=None, security=False):
     new_bug_ids = sorted(list(new_bug_ids))
 
     total_downloaded = 0
+    pbar = tqdm(total=len(new_bug_ids))
```

Comment:
You are not closing the `pbar`.
Maybe use `with`, so you can avoid closing it manually.